scripts/config_template.yaml

```diff
@@ -1,17 +1,15 @@
 # The data sets to load and how to interpret them
 data:
   data_set_1:
-    path: PATH_TO_DATA_GOES_HERE
-    estimate_col: estimate
-    date_col: date
     group_cols:
       geography: region
```

Comment:
Renaming the group columns (e.g. "geography" -> "region" here) is no longer done (had been part of `select_columns` which is now gone). So we should update this part of the config to give the column names for grouping factors, but not new names (i.e. group_cols is just a list, not itself a dictionary). This will precipitate edits to `extract_group_names` in __init__.py also - see below.

Reply:
I made a small change, but I'll leave the reworking of the pipeline for another PR